module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,9 @@
+#include "sort.hpp"
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> words){
+    words.sort();
```

Comment:
Will sort alone work? The goal of homework was to sort by length,then alphabetically. I'm afraid that sort alone will just sort all alphabetically.